mesa/mesaPROTON_OC.py

```diff
@@ -82,9 +87,9 @@ def __init__(self):
         self.num_oc_families = 8
         self.education_modifier = 1.0
 
-        # todo: Adding a statement for city selection
         # Folders definition
-        self.cwd = os.getcwd()
+        self.mesa_dir = os.getcwd()
+        self.cwd = self.mesa_dir[:-5]
```

Comment:
this sounds like magic. Is it os-independent? Why -5? Don't they have a name?

Reply:
It's true, that seems unclear.